assets/favicons/browserconfig.xml

```diff
@@ -0,0 +1,8 @@
+<browserconfig>
+    <msapplication>
+        <tile>
+            <square150x150logo src="/docs/5.2/assets/favicons/mstile-150x150.png"/>
```

Comment:
Don't know yet how to tackle that but `/docs/5.2/assets/favicons/mstile-150x150.png` is a reference to Boosted documentation.